drivers/bridge/bridge.go

```diff
@@ -674,6 +674,8 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 	bridgeAlreadyExists := bridgeIface.exists()
 	if !bridgeAlreadyExists {
 		bridgeSetup.queueStep(setupDevice)
+	} else {
+		bridgeSetup.queueStep(ensureIsBridgeInterface)
```

Comment:
Minor comment. Maybe we can check in the previous func `newInterface(nlh *netlink.Handle, config *networkConfiguration) *bridgeInterface` and make it also return an error?